src/script/interpreter.h

```diff
@@ -24,6 +26,10 @@ enum
     SIGHASH_NONE = 2,
     SIGHASH_SINGLE = 3,
     SIGHASH_ANYONECANPAY = 0x80,
+
+    SIGHASH_TAPDEFAULT = 0,
```

Comment:
Use taproot instead of tap?
SIGHASH_TAPROOT_DEFAULT
TAPROOT like other instances, e.g. TAPROOT_PROGRAM_SIZE.

Also, SignatureHashTap() renamed to SignatureHashTaproot()?
Similar to VerifyTaprootCommitment.

Reply:
Agree, those enum values were weird. Changed to SIGHASH_INPUT_MASK, SIGHASH_OUTPUT_MASK, and SIGHASH_DEFAULT; even though they're Taproot specific for now, that's not inherently the case.

`SignatureHashTap` wasn't actually necessary, it was a leftover from older code. I've removed it.